pkg/authorization/authorizer/attributes.go

```diff
@@ -21,7 +21,7 @@ type DefaultAuthorizationAttributes struct {
 
 // ToDefaultAuthorizationAttributes coerces Action to DefaultAuthorizationAttributes.  Namespace is not included
 // because the authorizer takes that information on the context
-func ToDefaultAuthorizationAttributes(in authorizationapi.Action) DefaultAuthorizationAttributes {
+func ToDefaultAuthorizationAttributes(in authorizationapi.Action) Action {
 	return DefaultAuthorizationAttributes{
```

Comment:
huh... this is a chunky struct to copy around (either as an arg or as a receiver). not new, but worth making a pointer in a follow up.